docs/assignment1.md

```diff
@@ -1,62 +1,75 @@
+
+With the game briefly explained, we must now consider how to develop this idea in a way that is easily extendable, maintainable and comprehensible. By using the Object-Oriented programming paradigm, we aim to create a system that is easily modifiable, expandable and intuitive for programmers to understand. To help achieve this goal, we will be making use of UML diagrams. 
 
-<When defining both functional features and quality requirements, remember that you will need to come back to them in Assignments 2 and 3 and explicitly state how specific parts of models/implementation satisfy them.>
+   				   Figure 1: A use case diagram of VUORK
+
+![VuORK use case diagram](https://lh3.googleusercontent.com/ePg0EoRTAvMiKXEP_VNm-gGz7HN0pBql_kOfZ-RUjralgX5VEYfBDSGg7DdKa8HeYwpv4C79SSpxdnjOcfLk6GLn7E-IDENFk6257sC3hyDslfFupreykPBXSfHrvP1NY3T5XnxJ)
+
+The remainder of this paper will cover detailed features and quality requirements to help us structure this project (This should be self explanatory). 
+
+## Features
+Author(s): Marta Jansone, Theresa Schantz
 
 ### Functional features
 
 As a preamble to the table, you can discuss the main line of reasoning you applied for defining the functional features provided in the table.
 
-| ID  | Short name  | Description  |
-|---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+| ID   | Short name |                         Description                          |
+| ---- | ---------- | :----------------------------------------------------------: |
+| F1   | Commands   | The player can move the character using the following commands: *"move, take, put, drop, look, attack"*. The structure of this command will be <action> <items or objects>. This will help easily parse the input and help keep the commands easily comprehensible for the user. The command may include such articles as “the”, “a” or “an” and such prepositions as, for example, “in“ or “at”. |
+|      |            |                                                              |
+| F2   | Movements  | The player can move around in the game environment and get from one room to another by using the following commands: *"go/move followed by north, south, west, east, down, up, left and right"* |
+|      |            |                                                              |
+| F3   | Narrator   | Computer responding to user; describing what is happening, what the user is able to see |
```

Comment:
A more in depth explanation of this feature is needed, how and in what cases does this narration occur? What is meant by ' user is able to see'. Think about how the limitation in vision may be implemented